package.json

```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "jest": { "preset": "ts-jest" },
   "dependencies": {
-    "lodash.at": "4.6.0",
+    "lodash": "4.17.15",
```

Comment:
Shouldn't dependencies be a bit less strict so that it doesn't force a specific verion on the user of the lib?

Reply:
You are right. Changing it

Reply:
do not bloat factorial-i18n with underscore just for `isEmpty.

```jsx
const isEmpty = (obj) => Object.keys(obj).length === 0
```

Reply:
I was not requiring the whole lodash, I'm requiring `at` and `isEmpty` only. It is almost the same as you did.

Reply:
will tree-shacking work? I remember some time ago it was not working. Please check the output code.

Reply:
Sure, I can do this

Reply:
Another reason is because the `lodash.at` and `lodash.isEmpty` are not updated anymore.

Reply:
Did you use `https://www.npmjs.com/package/webpack-bundle-analyzer` to do it?